host/volume/zfs/integrations.go

```diff
@@ -0,0 +1,32 @@
+package zfs
+
+import (
+	"fmt"
+	"strings"
+
+	gzfs "github.com/flynn/flynn/Godeps/_workspace/src/github.com/mistifyio/go-zfs"
+)
+
+type Logger struct{}
+
+func (*Logger) Log(msg []string) {
+	fmt.Printf("[zfs] %s\n", strings.Join(msg, " ")) // TODO replace with log15
```

Comment:
How should we tag output and at what level?  This gets called for every zfs command go-zfs invokes, so probably trace... I'd like to have a way to tag it with which provider invoked it but I don't see a ready way to set a logger per provider without doing some very janky mutexes and "context" objects.

Alternatively, we may not have much need for this at all in the final code.